client/src/styles/minSummary.css

```diff
@@ -25,7 +27,7 @@
 }
 
 .minBookImage{
-    width: 60%;
+    height: 110px;
```

Comment:
is this responsive?

Reply:
technically no, as in it won't scale with screen size, but that was the planned behavior. Wanted the book images to look the same same size on either screen (don't want it to be too big or too small). Try viewing it with chrome mobile and let me know if it looks good.